src/rulesets/oas3/__tests__/valid-example.ts

```diff
@@ -3,6 +3,8 @@ import { oas3Rules } from '../index';
 
 const ruleset = { rules: oas3Rules() };
 
+declare var test: jest.It;
```

Comment:
Why is this required?

Reply:
We have `mocha` test runner included in Spectral and TS is confused and doesn't know which `it` function it should refer to.

Reply:
Can we get rid of mocha? I think oclif added jest support a while back. https://github.com/oclif/oclif/pull/219

Reply:
Sorry, was wrong. Mocha is not listed as a dependency, but TS still sees typings for it and prefers it over Jest.

Reply:
```
# npm ls @types/mocha # gives the following
@stoplight/spectral@0.0.0 /home/Stoplight/spectral
└─┬ @oclif/test@1.2.4
  └─┬ fancy-test@1.4.3
    └── @types/mocha@5.2.6 
```
Seems like oclif/test requires types/mocha

Reply:
Well, they haven't released any newer version, so we'll need to stick to that unless you have a better workaround.
The reason I had to override the type is caused by the use of `test.each`. Plain `it` or `test` have a very similar, if not exact, signature.
Should I add a comment explaining that?
@XVincentX @philsturgeon

Reply:
Ok figured it out in person. We can't do anything about this unless we decide to ditch the whole oclif testing framework.